### PR TITLE
Hard limit for S3 artifact upload size

### PR DESCRIPTION
**Pull request by karensawrey**
As per title. A tiny addition as a result of responding to a ticket.

### CODE REVIEW

File: pages/agent/v3/cli_artifact.md
Comment (gilesgas) [inline pages/agent/v3/cli_artifact.md:13]: Re-writing this on a single line...

```suggestion
You can use this command in your build scripts to store artifacts. Artifacts are accessible using the web interface and can be downloaded by future build steps.
```

File: pages/agent/v3/cli_artifact.md
Comment (gilesgas) [inline pages/agent/v3/cli_artifact.md:14]: Thanks @karensawrey ,

Could you please confirm if the 5Gb limit relates to the size of a single artifact, or is it multiple artifacts totalling 5Gb in a single upload?

I'm assuming you mean the former and I'll suggest the following re-wording of this line to clarify that. However, I'll need confirmation about my question above before approving this PR, because if it's the latter, I'll need to change the wording again.

```suggestion
Artifacts can be stored in the Buildkite-managed artifact store, or your own storage location, depending on how you have configured your Buildkite Agent.

Be aware that the Buildkite-managed artifact store has an upload size limit of 5Gb for per artifact.
```

Comment (gilesgas) [reply pages/agent/v3/cli_artifact.md:14]: Actually, I confirmed the answer to my question above in the related ticket.

If this is all good with you, please accept my suggestions before merging, and I'll approve this PR in the meantime.

**Review by gilesgas — APPROVED** [review]

**Comment by karensawrey** [comment]
Thanks for looking and touching it up, @gilesgas! merging after the tests pass!

**Pull request merged**
